api/indexer.oas2.json

```diff
@@ -945,6 +991,28 @@
         }
       }
     },
+    "ApplicationLogData": {
+      "description": "Stores the global information associated with an application.",
+      "type": "object",
+      "required": [
+        "txid",
+        "logs"
+      ],
+      "properties": {
+        "txid": {
+          "description": "Transaction ID",
```

Comment:
Do you think we should specify that this is the "parent" txid?

Thinking out loud about the next bit of work... If we were to add inner transactions to the txn table, we might run into issues with logs being double reported. For example, suppose an inner transaction generates logs and we search for logs on that round. We need to make sure that when we match the parent and the child we only print the logs once.

Reply:
Current implementation does not look up recursively - it is more about the implementation and not API.
About the `txid`, as discussed, it is an implicit parent id since only regular txns have ids, and specifying "parent" word looks a bit confusing since logs can exist even without inner txns